test/compatibility/compatibility.test.ts

```diff
@@ -99,15 +100,11 @@ describe('Backwards and forwards copmatibility', () => {
     // Prints as 'Helloøã'
     const expected = decodeSafe64('SGVsbG_44wA=');
 
-    expect(decrypted).toEqual(expected);
+    expect(util.createBuffer(decrypted as Uint8Array).data).toEqual(expected);
```

Comment:
This typecast sees wrong. The result of `decryptStringWithKey` should be a string. If this is working I think this is a bug?